### ISSUE TITLE
Incompatible with systemd-190 + json output

### ISSUE THREAD

**Issue by SjonHortensius** [systemd/journal2gelf#2]
Since systemd-190 ( http://lists.freedesktop.org/archives/systemd-devel/2012-September/006625.html ) the json output format has changed, your parser still expects [ .. ], but systemd now outputs single objects per line

**Comment by joemiller** [comment]
Yes, this change in the format should actually make the parser in journal2gelf significantly simpler and was a change we hoped would happen. However, I don't have any boxes running systemd-190 at the moment so I have no easy way or priority to test. It looks like systemd-190 may be included in fedora-18 which means we would need to make this change soon. I will do this as soon as possible. Patches are also welcome

**Comment by SjonHortensius** [comment]
I did a few qnd attempts, but cannot get it to work due to my limited python skills. An example of the output:

```
{ "__CURSOR" : "...", "__REALTIME_TIMESTAMP" : "1370610301339049", "__MONOTONIC_TIMESTAMP" : "6439864", "_BOOT_ID" : "c8464d9f4bc94f8b83b9295a545c82b4", "PRIORITY" : "6", "_MACHINE_ID" : "...", "_HOSTNAME" : "...", "_SOURCE_MONOTONIC_TIMESTAMP" : "0", "_TRANSPORT" : "kernel", "SYSLOG_IDENTIFIER" : "kernel", "MESSAGE" : "efi: mem05: type=2, attr=0xf, range=[0x0000000002000000-0x0000000002f51000) (15MB)" }
{ "__CURSOR" : "...", "__REALTIME_TIMESTAMP" : "1370610301339058", "__MONOTONIC_TIMESTAMP" : "6439872", "_BOOT_ID" : "c8464d9f4bc94f8b83b9295a545c82b4", "PRIORITY" : "6", "_MACHINE_ID" : "...", "_HOSTNAME" : "...", "_SOURCE_MONOTONIC_TIMESTAMP" : "0", "_TRANSPORT" : "kernel", "SYSLOG_IDENTIFIER" : "kernel", "MESSAGE" : "efi: mem06: type=7, attr=0xf, range=[0x0000000002f51000-0x0000000020000000) (464MB)" }
```

Ps. I also created a go variant inspired by your script @ https://github.com/react-com/SystemdJournal2Gelf

**Comment by SjonHortensius** [comment]
It seems you have fixed this issue in e9f6a81
